.travis.yml

```diff
@@ -22,20 +22,27 @@ matrix:
       env: DB=MYSQL PDO=1 PHPUNIT_COVERAGE_TEST=1
 
 before_script:
-  - composer validate
-  - composer self-update || true
-  - "if [ \"$PHPCS_TEST\" = \"1\" ]; then pyrus install pear/PHP_CodeSniffer; fi"
+# Init PHP
+  - export CORE_RELEASE=$TRAVIS_BRANCH
```

Comment:
We don't need to keep adding in CORE_RELASE env anymore. Shall we drop this in the future? It was only necessary for travis-support-module.